server/user_account/views.py

```diff
@@ -110,12 +110,10 @@ def remove_token(self, request): # pylint: disable=no-self-use
 
 class AccessAllClients(generics.GenericAPIView):
     queryset = CustomUser.objects.values(
-        'user_name',
         'first_name',
         'last_name',
         'role',
         'is_active',
-        'email',
         )
     http_method_names = ['get']
```

Comment:
You should refactor that view:
1- Don't hard code the value that you want to pass through this endpoint inside the queryset, you should create a serializer for that.
2- No need to specify http_method_names = ['get'] because you are using a GenericAPIView and you are already specifying in your get method definition that the operation that will only be allowed is a get.
3-You must secure this endpoint( Please check the last video I did on authentication under Sprint2/ in the drive or read the doc here https://www.django-rest-framework.org/api-guide/permissions/ )
5- Also you might want to add pagination, for now its okay but later when we will have a huge db we dont want to return all the query at once. (optional for now)
4-For now keep this method but modify the changes mentioned above, also this will probably be deleted because the viewset responsible for registration already implement this functionality (it handles CRUD) so we will just rename the registrationViewSet to UserViewSet) just to explain the future vision.

Reply:
2) Addressed in commit b9d15ea4438210143bedf032d85ad6b55e2ce729

Reply:
5- Addressed in commit ef691e7f16ed2ed697ba24007f9fd40a58b457eb

Reply:
4- I have no idea why this is being commented, please keep review comments to requested changes

Reply:
4- Since you worked on this method and you did a change on it i’m just explaining the future vision of that since i was adding comments on it.

Reply:
1. Fixed as of commit d3e6a3ed4f19148693879d75a0b8c0de80072ddb

Reply:
3. Fixed as of commit: beaa57e70e4afaf1bee491c6f27c0196e85ca8ed